### ISSUE TITLE
backup is not  compatible with 7.x

### ISSUE THREAD

**Issue by tkoeberl** [guilhemmarchand/trackme#267]
**Describe the bug**
Using in 7.3.2.
```
| trackme url="/services/trackme/v1/backup_and_restore/backup" mode="post"
```
results in:
```
response: Warn: exception encountered: 'newline' is an invalid keyword argument for this function
```

**To Reproduce**
call custom search command

**Expected behavior**
does the job

**Splunk version and deployment:**
7.3.2. Clustered SH

**Additional context**
Splunkbase still saying it is for 7.x
https://splunkbase.splunk.com/app/4621/

**Comment by guilhemmarchand** [comment]
Nice finding @tkoeberl ! Thanks
Must be an incompaibility thing between Python2 and Python3, will check!

**Comment by guilhemmarchand** [comment]
newline function used with csv module is not compatible with Python 2, the following patch will fix:

```
            # Get the Kvstore collections to be backed up from a lookup stored in the app
            if sys.version_info.major == 2:

                try:
                    with open(os.path.join(splunkhome, 'etc', 'apps', 'trackme', 'lookups', 'trackme_kvstore_collections.csv')) as f:
                        reader = csv.reader(f)
                        collection_list = list(reader)

                except Exception as e:
                    return {
                        'payload': 'Warn: exception encountered: ' + str(e) # Payload of the request.
                    }

            else:

                try:
                    with open(os.path.join(splunkhome, 'etc', 'apps', 'trackme', 'lookups', 'trackme_kvstore_collections.csv'), newline='') as f:
                        reader = csv.reader(f)
                        collection_list = list(reader)

                except Exception as e:
                    return {
                        'payload': 'Warn: exception encountered: ' + str(e) # Payload of the request.
                    }
```

**Comment by guilhemmarchand** [comment]
@tkoeberl 
Fixed in 1.2.36 published in Splunk Base.
